virt/node_maintenance/virt-using-skip-node.adoc

```diff
@@ -0,0 +1,13 @@
+[id="virt-using-skip-node"]
+= Using skip node annotation
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-using-skip-node
+
+toc::[]
+
+include::modules/virt-using-skip-node.adoc[leveloffset=+1]
+
+.Additional resources:
```

Comment:
no `:`

Reply:
Because this is in an assembly, it needs a `==` heading instead of the `.` and it also needs an anchor ID.

[Info about assembly/module titles and section headings](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings)